species/Drake.coffee

```diff
@@ -132,22 +131,21 @@ BioLogica.Species.Drake =
 
     "color":
       "Frost":    [["c","c"]]
-      "Steel":    BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["B"],["B","b"],["D"],["D","d","dl","Y"]])
-      "Copper":   BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["b"],["b"],["D"],["D","d","dl","Y"]])
-      "Silver":   BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["B"],["B","b"],["d","dl"],["d","dl","Y"]])
-      "Gold":     BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["b"],["b"],["d","dl"],["d","dl","Y"]])
-      "Charcoal": BioLogica.combinations([["C"],["C","c"],["m"],["m"],["B"],["B","b"],["D"],["D","d","dl","Y"]])
-      "Lava":     BioLogica.combinations([["C"],["C","c"],["m"],["m"],["b"],["b"],["D"],["D","d","dl","Y"]])
-      "Ash":      BioLogica.combinations([["C"],["C","c"],["m"],["m"],["B"],["B","b"],["d","dl"],["d","dl","Y"]])
-      "Sand":     BioLogica.combinations([["C"],["C","c"],["m"],["m"],["b"],["b"],["d","dl"],["d","dl","Y"]])
+      "Steel":    BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["B"],["B","b"],["D"],["D","d","Y"]])
+      "Copper":   BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["b"],["b"],["D"],["D","d","Y"]])
+      "Silver":   BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["B"],["B","b"],["d"],["d","Y"]])
+      "Gold":     BioLogica.combinations([["C"],["C","c"],["M"],["M","m"],["b"],["b"],["d"],["d","Y"]])
+      "Charcoal": BioLogica.combinations([["C"],["C","c"],["m"],["m"],["B"],["B","b"],["D"],["D","d","Y"]])
+      "Lava":     BioLogica.combinations([["C"],["C","c"],["m"],["m"],["b"],["b"],["D"],["D","d","Y"]])
+      "Ash":      BioLogica.combinations([["C"],["C","c"],["m"],["m"],["B"],["B","b"],["d"],["d","Y"]])
+      "Sand":     BioLogica.combinations([["C"],["C","c"],["m"],["m"],["b"],["b"],["d"],["d","Y"]])
 
     "health":
       "Bog breath": [['bog','bog'],['bog','Y']]
       "Healthy": [['Bog', 'Bog'],['Bog','bog'],['Bog','Y']]
 
     "liveliness":
```

Comment:
I think that liveliness can be removed as a trait, if `dl` is gone. This species file is already not going to be used by any application that would want to use liveliness, since these orgs can't be dead through genetics.